conf/zeppelin-site.xml.template

```diff
@@ -61,6 +61,12 @@
   <description>path or URI for notebook persist</description>
 </property>
 
+<property>
+  <name>zeppelin.note.file.exclude.fields</name>
+  <value>results,status</value>
```

Comment:
The default value should be empty, because users would copy `zeppelin-site.xml.template` to `zeppelin-site.xml`

Reply:
Comment addressed